### PR TITLE
Add PID results display table

### PR DESCRIPTION
**Pull request by bondjimbond**
Adds the object PID to the table of results.

### CODE REVIEW

File: includes/replace.form.inc [hunk 1]
```diff
@@ -31,6 +31,8 @@ function islandora_find_replace_replace_form($form, &$form_state, $find_replace)
 
   foreach (unserialize($find_replace['find_results']) as $option) {
     $options[$option['object']] = array(
+      'pid' => l($option['object'], 'islandora/object/' . $option['object'],
```
Comment (DiegoPino) [inline includes/replace.form.inc]: Hi! Nice. Do we need both to have links to the same?

Comment (DiegoPino) [reply includes/replace.form.inc]: What i mean is maybe we just need a single link. Does not change anything other than user experience.

Comment (bondjimbond) [reply includes/replace.form.inc]: Hard to say. I personally look for links under the PID, but others might go for the title instead. Any reason it's better not to have it in both places?

**Comment by bondjimbond** [comment]
Side question: why are all the improvements in the `7.x-regex` branch and not in `master`?

File: includes/replace.form.inc (hunk 1 above)
Comment (DiegoPino) [reply includes/replace.form.inc]: I would say there is no real huge reason but i don't it confuses people to have two links to the same. Up to you really. As long as we have a link i'm ok with it!

**Comment by DiegoPino** [comment]
@bondjimbond just in case people still like the boring master one? 😆  But i'm totally for merging regex into master once we get this improvement of yours merged. Does that sound good?

File: includes/replace.form.inc (hunk 1 above)
Comment (bondjimbond) [reply includes/replace.form.inc]: I'm happier to leave it as two links if you don't mind! :)

**Comment by bondjimbond** [comment]
Sounds good to me!

**Pull request merged**